python/cudf/cudf/core/dataframe.py

```diff
@@ -2602,6 +2602,71 @@ def insert(self, loc, name, value, nan_as_null=None):
 
         self._data.insert(name, value, loc=loc)
 
+    def diff(self, periods=1, axis=0):
+        """
+        First discrete difference of element.
+
+        Calculates the difference of a Dataframe element compared with another
+        element in the Dataframe (default is element in previous row).
+
+        Parameters
+        ----------
+        periods : int, default 1
+            Periods to shift for calculating difference,
+            accepts negative values.
+        axis : {0 or ‘index’, 1 or ‘columns’}, default 0
```

Comment:
Avoid smart quotes. These are [straight quotes in the original pandas docstring](https://github.com/pandas-dev/pandas/blob/42a26ac539ac9a194f6e7da0897ffc938b7d7e76/pandas/core/frame.py#L8450
) and Sphinx converts them to curly (smart) quotes in the browser. 
```suggestion
        axis : {0 or 'index', 1 or 'columns'}, default 0
```